packages/ivass-certified-attributes-importer/aws.config.local

```diff
@@ -0,0 +1,4 @@
+[default]
+aws_access_key_id=test-aws-key
+aws_secret_access_key=test-aws-secret
```

Comment:
This could not work if your service using S3(minio), docker-compose for local E2e testing provides a minio instance with differents access id key and secret, see [here](https://github.com/pagopa/interop-be-monorepo/blob/main/docker/docker-compose.yml#L117) the new values.